circle.yml

```diff
@@ -12,6 +12,7 @@ dependencies:
     - sudo apt-get update; sudo apt-get install python-opencv libsuitesparse-dev
     - sudo cp /usr/lib/python2.7/dist-packages/cv2.so /usr/local/lib/python2.7/dist-packages; sudo cp /usr/lib/python2.7/dist-packages/cv.py /usr/local/lib/python2.7/dist-packages
     - rake install_style_config
+    - pip install -U 'numpy>=1.10.1' 'scipy>=0.14.1'
```

Comment:
Since higher versions exist, just `pip install -U numpy scipy` has the same effect and I think is a little easier to parse. But no stress.